paperweight-lib/src/main/kotlin/tasks/GenerateReobfMappings.kt

```diff
@@ -169,11 +175,45 @@ abstract class GenerateReobfMappings : JavaLauncherTask() {
         }
     }
 
+    class RemoveRecompiledThisMappings(private val recompiledClasses: Set<String>) : ChangeContributor {
+        override fun contribute(
+            currentClass: ClassData?,
+            classMapping: ClassMapping<*, *>?,
+            context: HypoContext,
+            registry: ChangeRegistry
+        ) {
+            if (currentClass == null || classMapping == null) return
+            val outers = currentClass.outerClasses()
+            if (outers.isEmpty()) return
+            if (currentClass.isStaticInnerClass) return
+            if (outers[0].name() !in recompiledClasses) return
```

Comment:
I know it'll make this bigger, but please use `{ ... }` in all cases - only exception is `if` statements used as a ternary.